shell/platform/windows/client_wrapper/include/flutter/flutter_view_controller.h

```diff
@@ -43,8 +44,10 @@ class FlutterViewController {
   // either using CoreWindow.
   //
   // |dart_project| will be used to configure the engine backing this view.
```

Comment:
Please document the new argument, and mention that it can be null.

Reply:
Fixed in next commit.